client/src/components/WorkflowInvocationState/InvocationMessage.vue

```diff
@@ -175,38 +196,80 @@ const infoString = computed(() => {
         return reason;
     }
 });
+
+function openJobInNewTab(jobId: string) {
+    const url = `/jobs/${jobId}/view`;
+    window.open(url, "_blank");
+}
 </script>
 
 <template>
     <div>
         <div :class="levelClass" style="text-align: center">
             {{ infoString }}
         </div>
-        <div v-if="dependentWorkflowStep">
-            Problem occurred at this step:
-            <WorkflowInvocationStep
-                :invocation="invocation"
-                :workflow="workflow"
-                :workflow-step="dependentWorkflowStep"></WorkflowInvocationStep>
-        </div>
-        <div v-if="workflowStep">
-            {{ stepDescription }}
-            <WorkflowInvocationStep
-                :invocation="invocation"
-                :workflow="workflow"
-                :workflow-step="workflowStep"></WorkflowInvocationStep>
-        </div>
-        <div v-if="HdaId">
-            This dataset failed:
-            <GenericHistoryItem :item-id="HdaId" item-src="hda" />
-        </div>
-        <div v-if="HdcaId">
-            This dataset collection failed:
-            <GenericHistoryItem :item-id="HdcaId" item-src="hdca" />
-        </div>
-        <div v-if="jobId">
-            This job failed:
-            <JobInformation :job_id="jobId" />
+        <div class="invocation-error-grid d-flex flex-wrap">
+            <GCard
+                v-if="dependentWorkflowStep"
+                clickable
+                :current="activeNodeId === dependentWorkflowStep.id"
+                grid-view
+                @click="emit('view-step', dependentWorkflowStep.id)">
+                Problem occurred at this step:
+                <strong>
+                    <WorkflowStepTitle
+                        :step-index="dependentWorkflowStep.id"
+                        :step-label="
+                            dependentInvocationStep?.workflow_step_label || `Step ${dependentWorkflowStep.id + 1}`
+                        "
+                        :step-type="dependentWorkflowStep.type"
+                        :step-tool-id="dependentWorkflowStep.tool_id || undefined"
```

Comment:
Any reason not to instead accept null on the WorkflowStepTitle props ?

Reply:
Sorry, I would get these client errors for setting a vue component prop as `stepToolId?: string | null;` but turns out it was a false flag, fixed this now.